client/components/layout/main-layout/footer/Footer.tsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
-
+import { Link } from 'react-router-dom';
+//TODO: Add Links
 const Footer = () => {
-  return <div>Footer Placeholder</div>;
+  return (
+    <>
+      <div className="left-content">
```

Comment:
Please refer to figma file for correct spelling/formatting

Reply:
Updated based on your comments. Besides that, I checked several websites and think the first character should be © (copy right), not @.  Could you revisit the figma file to fix it?

Reply:
Also along with the latest commit, I'm still seeing typos. "Cookie Preferences" is a set term
@philzmintmojito thank you for pointing it out. I will go change on the figma now.
We definitely need more help with reviewing our mockup designs, but please also kindly note that all symbols are for mockups and many are just placeholders for faster churn-outs at the moment. To avoid confusion, would greatly appreciate all figma comments can be left directly on the figma file/file/element in the future.

Reply:
Thanks, Chen. Typo is fixed.